Images_And_Graphics_Index1.html

```diff
@@ -183,19 +105,21 @@ <h2>Information</h2>
 
 			<br>It is essential that all students on the trip play their part in risk management. Their personal safety is very much their responsibility, and they must always act in a manner that does not endanger either themselves or others. <br>It is important that they follow the instructions of 			 the staff at all times.<br>
 
-			<br>All these activities are compulsory but have been designed to be both educational and fun. Students will be supervised at all times by Tawa College staff. If you have not already returned the permission forms given out earlier this term, <br>then please collect another copy of the forms               from the school office and hand them in to the student office along with $75.00 before Friday, October 25. If you have already returned the permission forms, then you only need to pay $75.00 before Friday, October 25.<br></p></div>
-				  
-
+			<br>All these activities are compulsory but have been designed to be both educational and fun. Students will be supervised at all times by Tawa College staff. If you have not already returned the permission forms given out earlier this term, <br>then please collect another copy of the forms               from the school office and hand them in to the student office along with $75.00 before Friday, October 25. If you have already returned the permission forms, then you only need to pay $75.00 before Friday, October 25.<br></p></div>		  
 	</div>
 			
+	
 				
 			
 					</div>	
-			  <button onclick="topFunction()" id="myBtn" title="Go to top">TOP</button>
+			
+			<div class="box4"><img class="KORU_DESIGN" src="Images/19Korudesign.jpg" alt="iMages"></div>

```

Comment:
I added a box 4 where I added a koru pattern then added the back to top button as there was lots of white space below the text so I added a koru design there.